src/appleseed/renderer/kernel/lighting/pathvertex.h

```diff
@@ -72,6 +73,17 @@ class PathVertex
     const EDF*                  m_edf;
     const BSDF*                 m_bsdf;
     const void*                 m_bsdf_data;
+
+    // BSSRDF data
+    const BSSRDF*               m_bssrdf;
+    const void*                 m_bssrdf_data;
```

Comment:
You usually can't store and pass around data pointers since storage for this data is the stack (in the `InputEvaluator`).